src/workloads/execution/ProjectSort.yml

```diff
@@ -0,0 +1,152 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  Check performance of a project followed by a sort that can be swapped in order to push down. Check the
+  performance both without and without an index.
+  Important Metrics: timers.dur, timers.total on the aggregate commands. The index build is to test the performance
+  of the aggregation with/without an index.
+Keywords:
+  indexes
+  collections
+  aggregation
+  sort
+
+GlobalDefaults:
+  aggregate: &collName Collection0
+  pipeline: &aggPipe [{$project: {array: 0}}, {$sort: {integer: 1}}]
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: &docCount 100000
+    BatchSize: *docCount
+    Document:
+      integer: &integer {^RandomInt: {min: 1, max: 10000}}
+      double: &double {^RandomInt: {distribution: geometric, p: 0.1}}
+      string: &string {^RandomString: {length: 16}}
+      array:
+      - *integer
+      - *integer
+      - subInteger: *integer
+        subString: *string
+        subArray:
+        - *integer
+        - *integer
+      - subInteger: *integer
+        subString: *string
+        subArray:
+        - *integer
+        - *integer
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: BuildIndex
+  Type: RunCommand
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        createIndexes: *collName
+        indexes: &IndexSpec [{key: {"integer": 1}, name: "int_1"}]
+  - *Nop
+  - *Nop
+
+
+- Name: Quiesce
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
```

Comment:
Can you change this to use the QuiesceActor  (see either of the following examples: [1](https://github.com/mongodb/genny/blob/fd3a0f5f26bff7e13037850595a16f07f697b442/src/workloads/transactions/LLTAnalytics.yml#L236-L245) [2](https://github.com/mongodb/genny/blob/c38a522e44a12fec43558527f110998eaa817465/src/workloads/scale/TimeSeriesSortScale.yml#L41-L49)).

Reply:
Done